### ISSUE TITLE
Fix false positives for multiple pseudo-class in selector-pseudo-class-no-unknown

### ISSUE THREAD

**Issue by Mottie** [stylelint/stylelint#2810]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

When multiple pseudoclasses (chained) are added to the webkit scrollbar selector, we see a "selector-pseudo-class-no-unknown" error. In this example css:

```css
::-webkit-scrollbar-button:horizontal:decrement {
  background-color: transparent;
}

::-webkit-scrollbar-button:decrement:horizontal {
  background-color: transparent;
}

::-webkit-scrollbar-button:horizontal {
  background-color: transparent;
}

::-webkit-scrollbar-button:decrement {
  background-color: transparent;
}
```

The first two selectors show the error:

    1:38 error Unexpected unknown pseudo-class selector ":decrement" (selector-pseudo-class-no-unknown)
    5:37 error Unexpected unknown pseudo-class selector ":horizontal" (selector-pseudo-class-no-unknown)

The standard stylelint configuration is being used, and the error can be seen after entering the above css into the [main stylelint demo](https://stylelint.io/demo/).

> Which version of stylelint are you using?

`8.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Reproducible in the online demo

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

No "selector-pseudo-class-no-unknown" error.

> What actually happened (e.g. what warnings or errors you are getting)?

"selector-pseudo-class-no-unknown" error.

**Comment by evilebottnawi** [comment]
@Mottie thanks for issue, seems it is bug

**Comment by ntwb** [comment]
Fixed via https://github.com/stylelint/stylelint/commit/982024dd03380338d96bd1f7d48492708c87338d
